test/tables.dashdb.sql

```diff
@@ -1,40 +1,19 @@
-SET CURRENT SCHEMA = CURRENT USER;
```

Comment:
Does this file even _need_ to be changed and parameterized other than dropping this first line?